.github/workflows/release.yml

```diff
@@ -20,12 +20,12 @@ jobs:
   build:
     runs-on: ubuntu-20.04
     env:
-      BUILDROOT_VERSION: '2021.02.4'
+      BUILDROOT_VERSION: '2021.08.2'
```

Comment:
AFAIK the buildroot people are still working on the Pi2 support, shall we wait for their stable release?